opengever/api/mail.py

```diff
@@ -42,3 +44,13 @@ def __call__(self, validate_all=False, data=None, create=False):
             initialize_metadata(context, None)
 
         return context
+
+
+@implementer(ISerializeToJson)
+@adapter(IMail, Interface)
+class SerializeMailToJson(SerializeDocumentToJson):
+
+    def __call__(self, *args, **kwargs):
+        result = super(SerializeMailToJson, self).__call__(*args, **kwargs)
+        result[u'attachments'] = self.context.get_attachments()
```

Comment:
```suggestion
        result[u'items'] = self.context.get_attachments()
```

See discussion here: https://github.com/4teamwork/opengever.core/pull/6462#discussion_r431034923

Reply:
I'd tend to disagree here, because they are not children of the E-mail. For comparison, in the case of a user, this would be analogous to his `addresses` or something like that.

Reply:
As discussed I'll leave this under `attachments`